packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/components/NftAssetCountdown.tsx

```diff
@@ -1,29 +1,32 @@
 import React, { useEffect, useState } from 'react'
+import { colors } from '@blockchain-com/constellation'
 import styled from 'styled-components'
 
+import { Text } from 'blockchain-info-components'
+
 import { EthText } from '.'
 
 const CountdownText = styled(EthText)`
-  font-size: 20px;
+  font-size: 24px;
+  line-height: 32px;
+`
+const GreyLetter = styled(Text)`
+  color: ${colors.grey400};
+  padding-right: 8px;
 `
-
 const NftAssetCountdown: React.FC<Props> = ({ countDownDate }) => {
   const [Countdown, setCountdown] = useState('')
 
   useEffect(() => {
     const updateCountdown = () => {
       const now = new Date().getTime()
       const duration = countDownDate - now
-      const days = Math.floor(duration / (1000 * 60 * 60 * 24))
-      const hours = Math.floor((duration % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60))
-      const minutes = Math.floor((duration % (1000 * 60 * 60)) / (1000 * 60))
-      const seconds = Math.floor((duration % (1000 * 60)) / 1000)
-      setCountdown(
-        () => `${days ? `${days} days` : ''} ${hours} hours ${minutes} minutes ${seconds} seconds`
-      )
+      const days = Math.floor(duration / (1000 * 60 * 60 * 24)).toString()
+      const hours = Math.floor((duration % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)).toString()
+      const minutes = Math.floor((duration % (1000 * 60 * 60)) / (1000 * 60)).toString()
+      setCountdown(() => `${days},${hours},${minutes}`)
```

Comment:
Can we do seconds only if days < 1?

Hide days if days < 1 and show seconds if days < 1